<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectsWidget</class>
 <widget class="QWidget" name="ProjectsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projects</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>321</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="workspaceLabel">
      <property name="text">
       <string>Workspace:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>319</number>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>640</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
